### PR TITLE
Preparing release-3.8 for merge to master.

### PR DESCRIPTION
**Pull request by kmuriki**
## Description of the Pull Request (PR):

Preparing for the final v3.8.0 release.

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/hpcng/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR, linted with `make check`  and tested this PR locally with a `make test`, and `make testall` if possible (see CONTRIBUTING.md).
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/hpcng/singularity/blob/master/CONTRIBUTORS.md)

### CODE REVIEW

File: .github/RELEASE_PROCEDURE.md [hunk 1]
```diff
@@ -1,33 +1,86 @@
...
+   branch e.g. `git checkout upstream/master -b release-3.8`.
+2. Push the release branch to GitHub via `git push upstream release-3.8`.
+3. Examine the GitHub branch protection rules, to extend them to the
+   new release branch if needed.
+4. Modify the `README.md`, `INSTALL.md`, `CHANGELOG.md` via PR against
+   the release branch, so that they reflect the version to be released.
+5. Apply an annotated tag via `git tag -a -m "Singularity v3.8.0
+   Release Candidate 1" v3.8.0-rc.1`.
+6. Push the tag via `git push upstream v3.8.0-rc.1`.
+7. Create a tarball via `mconfig -v && make dist`.
```
Comment (ikaneshiro) [inline .github/RELEASE_PROCEDURE.md]: Usually `mconfig` isn't in everyone's path. I always use `./mconfig`

File: .github/RELEASE_PROCEDURE.md [hunk 2]
```diff
@@ -1,33 +1,86 @@
...
+   version, branched, and tagged.
+  - [User Docs](https://singularity.hpcng.org/user-docs/master/) can be edited [here](https://github.com/hpcng/singularity-userdocs)
+  - [Admin Docs](https://singularity.hpcng.org/admin-docs/master/) can be edited [here](https://github.com/hpcng/singularity-admindocs)
+2. Ensure the user and admin documentation has been deployed to the
+   singularity.hpcng.org website.
+4. Modify the `README.md`, `INSTALL.md`, `CHANGELOG.md` via PR against
+   the release branch, so that they reflect the version to be released.
+5. Apply an annotated tag via `git tag -a -m "Singularity v3.8.0" v3.8.0`.
+6. Push the tag via `git push upstream v3.8.0-rc.1`.
+7. Create a tarball via `mconfig -v && make dist`.
```
Comment (ikaneshiro) [inline .github/RELEASE_PROCEDURE.md]: Same question here

**Review by ikaneshiro — COMMENTED** [review]
Nitpick, otherwise lgtm

File: .github/RELEASE_PROCEDURE.md (hunk 1 above)
Comment (kmuriki) [reply .github/RELEASE_PROCEDURE.md]: I do the same. Fixed.

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**